EveripediaNetwork/ep-ui
# Fixed the duplicate wiki in profile problem

# Fixed the duplicate wiki in profile problem

_This solves the duplication of wikis created in the user profile

## How should this be tested?
create a wiki and check your profile


1. Before
![Screenshot (240)](https://user-images.githubusercontent.com/75235148/172628103-c33618e9-46ab-4222-9fbf-1a5e2f1acc03.png)

 
![Screenshot (242)](https://user-images.githubusercontent.com/75235148/172628129-1f5d87dc-76b6-4bdc-9405-a897f8fc95a7.png)


2. After
![Screenshot (239)](https://user-images.githubusercontent.com/75235148/172628222-2f89f2e3-5f34-4402-b9a0-c7db8a8b1aad.png)




## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/417
